dkg/frost/mock_network.go

```diff
@@ -0,0 +1,100 @@
+package frost
```

Comment:
we don't use mocks in implementation, that's why we have the testing utils

Reply:
Will refactor the tests and use testing utils.